### PR TITLE
docs: svg as component example leveraging swc instead of babeljs

### PR DESCRIPTION
**Pull request by rubens-lopes**
## Documentation / Examples

- [x] Make sure the linting passes by running `yarn lint`

### CODE REVIEW

**Comment by balazsorban44** [comment]
I think we should just convert https://github.com/vercel/next.js/tree/canary/examples/svg-components instead of adding a new example. :thinking:

**Comment by leerob** [comment]
Agreed, let's do that!

**Comment by rubens-lopes** [comment]
While using this example with jest I've found out that we need to mock the SvgComponent so `testing-library` is able to render svgs, but I couldn't find an way to do that globally. May I include a sample together?

File: examples/svg-components/README.md [hunk 1]
```diff
@@ -1,23 +1,22 @@
...
 
 Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
 
 ```bash
-npx create-next-app --example svg-components svg-components-app
+npx create-next-app --example svg-components-with-swc svg-components-app
 # or
-yarn create next-app --example svg-components svg-components-app
+yarn create next-app --example svg-components-with-swc svg-components-app
 # or
```
Comment (ijjk) [inline examples/svg-components/README.md:19]:
```suggestion
# or
pnpm create next-app -- --example svg-components svg-components-app
```

File: examples/svg-components/README.md [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
 ## How to use
 
 Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
 
 ```bash
-npx create-next-app --example svg-components svg-components-app
+npx create-next-app --example svg-components-with-swc svg-components-app
 # or
-yarn create next-app --example svg-components svg-components-app
+yarn create next-app --example svg-components-with-swc svg-components-app
```
Comment (ijjk) [inline examples/svg-components/README.md:18]:
```suggestion
yarn create next-app --example svg-components svg-components-app
```

File: examples/svg-components/README.md [hunk 3]
```diff
@@ -1,23 +1,22 @@
...
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/svg-components&project-name=svg-components&repository-name=svg-components)
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/svg-components-with-swc&project-name=svg-components-with-swc&repository-name=svg-components-with-swc)
 
 ## How to use
 
 Execute [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app) with [npm](https://docs.npmjs.com/cli/init) or [Yarn](https://yarnpkg.com/lang/en/docs/cli/create/) to bootstrap the example:
 
 ```bash
-npx create-next-app --example svg-components svg-components-app
+npx create-next-app --example svg-components-with-swc svg-components-app
```
Comment (ijjk) [inline examples/svg-components/README.md:16]:
```suggestion
npx create-next-app --example svg-components svg-components-app
```

File: examples/svg-components/README.md [hunk 4]
```diff
@@ -1,23 +1,22 @@
...
+This example uses svg components without disabling SWC using @svgr/webpack to import`.svg` files and rendering them as React components. [@svgr/webpack](https://www.npmjs.com/package/@svgr/webpack).
 
 ## Deploy your own
 
-Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/svg-components)
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/svg-components-with-swc)
 
-[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/svg-components&project-name=svg-components&repository-name=svg-components)
+[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/svg-components-with-swc&project-name=svg-components-with-swc&repository-name=svg-components-with-swc)
```
Comment (ijjk) [inline examples/svg-components/README.md:9]:
```suggestion
[![Deploy with Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/vercel/next.js/tree/canary/examples/svg-components&project-name=svg-components&repository-name=svg-components)
```

File: examples/svg-components/README.md [hunk 5]
```diff
@@ -1,23 +1,22 @@
-# SVG components example
+# SVG components example with SWC
 
-This example uses a custom `.babelrc` to add support for importing `.svg` files and rendering them as React components. [babel-plugin-inline-react-svg](https://www.npmjs.com/package/babel-plugin-inline-react-svg) is used to handle transpiling the SVGs.
+This example uses svg components without disabling SWC using @svgr/webpack to import`.svg` files and rendering them as React components. [@svgr/webpack](https://www.npmjs.com/package/@svgr/webpack).
 
 ## Deploy your own
 
-Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/svg-components)
+Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/svg-components-with-swc)
```
Comment (ijjk) [inline examples/svg-components/README.md:7]:
```suggestion
Deploy the example using [Vercel](https://vercel.com?utm_source=github&utm_medium=readme&utm_campaign=next-example) or preview live with [StackBlitz](https://stackblitz.com/github/vercel/next.js/tree/canary/examples/svg-components)
```

File: examples/svg-components/README.md [hunk 6]
```diff
@@ -1,23 +1,22 @@
-# SVG components example
+# SVG components example with SWC
 
-This example uses a custom `.babelrc` to add support for importing `.svg` files and rendering them as React components. [babel-plugin-inline-react-svg](https://www.npmjs.com/package/babel-plugin-inline-react-svg) is used to handle transpiling the SVGs.
+This example uses svg components without disabling SWC using @svgr/webpack to import`.svg` files and rendering them as React components. [@svgr/webpack](https://www.npmjs.com/package/@svgr/webpack).
```
Comment (ijjk) [inline examples/svg-components/README.md:3]:
```suggestion
This example uses svg components without disabling SWC using @svgr/webpack to import `.svg` files and rendering them as React components. [@svgr/webpack](https://www.npmjs.com/package/@svgr/webpack).
```

**Review by ijjk — APPROVED** [review]
Thanks for the PR!

**Pull request merged**